keps/sig-node/2570-memory-qos/README.md

```diff
@@ -104,6 +107,62 @@ Brief map as follows:
 | pod | sum(requests.memory) | n |
 | node | n/a | pods, kube-reserved, system-reserved | n |
 
+#### Beta
+The formula for memory.high is modified in Beta version of the feature. It will be set based on formula:
+```
+memory.high=floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize, where default value of memory throttling factor is set to 0.9
+```
+
+e.g. If a container has `requests.memory=50, limits.memory=100`, and we have a throttling factor of .9, `memory.high` would be 90. If a container had no memory limit specified, we substitute `limits.memory` for `node allocatable memory` and apply the throttling factor of .9 to that value.
+
+Node reserved resources(kube-reserved/system-reserved) are either considered. It is tied to `--enforce-node-allocatable` and `memory.min` will be set properly.
+
+Brief map as follows:
+| type | memory.min | memory.high |
+| -------- | -------- | -------- | 
+| container | requests.memory | floor[(requests.memory + memory throttling factor * (limits.memory or node allocatable memory - requests.memory))/pageSize] * pageSize |
+| pod | sum(requests.memory) | n |
+| node | n/a | pods, kube-reserved, system-reserved | n |
```

Comment:
The last column is redundant. And there is something wrong with this table, as I have mentioned above. @ndixita Could you please fix it in both the alpha and beta chapters?